doc/UserGuide/CustomizingTheUPP/EnablingOutput.rst

```diff
@@ -0,0 +1,360 @@
+      <shortname>TMP_ON_SIGMA_LVLS</shortname>
+      <pname>TMP</pname>
+      <fixed_sfc1_type>sigma_lvl</fixed_sfc1_type>
+      <level>0. 500. 1000. 1500. 2000. 2500. 3000. 3500. 4000. 4500. 5000. 5500. 6000. 6500. 7000. 7500. 8000. 8500. 9000. 9500. 10000.</level>
+      <scale>4.0</scale>
+   </param>
+
+   <param>
+      <post_avblfldidx>208</post_avblfldidx>
+      <shortname>UGRD_ON_SIGMA_LVLS</shortname>
+      <pname>UGRD</pname>
+      <fixed_sfc1_type>sigma_lvl</fixed_sfc1_type>
+      <level>0. 500. 1000. 1500. 2000. 2500. 3000. 3500. 4000. 4500. 5000. 5500. 6000. 6500. 7000. 7500. 8000. 8500. 9000. 9500. 10000.</level>
+      <scale>4.0</scale>
+   </param>
+
+   <param>
+      <post_avblfldidx>209</post_avblfldidx>
+      <shortname>VGRD_ON_SIGMA_LVLS</shortname>
+      <pname>VGRD</pname>
+      <fixed_sfc1_type>sigma_lvl</fixed_sfc1_type>
+      <level>0. 500. 1000. 1500. 2000. 2500. 3000. 3500. 4000. 4500. 5000. 5500. 6000. 6500. 7000. 7500. 8000. 8500. 9000. 9500. 10000.</level>
+      <scale>4.0</scale>
+   </param>
+
+.. _validate-xml:
+
+Validate the XML postcontrol file
+-----------------------------------
+
+After modifying the XML postcontrol file, users must validate the file to ensure it was formatted successfully:
+
+.. code-block:: console
+
+   cd UPP/parm
+   # For GFS/Global
+   xmllint --noout --schema EMC_POST_CTRL_Schema.xsd gfs/postcntrl_gfs_f00_two.xml
+   # For LAM/regional
+   xmllint --noout --schema EMC_POST_CTRL_Schema.xsd rrfs/rrfs_postcntrl.xml
+
+If the XML file is formatted correctly, it will print a message like:
+
+.. code-block:: console
+
+   gfs/postcntrl_gfs_f00_two.xml validates
+
+.. _gen-text:
+
+Generate the flat text file
+----------------------------
+
+If the control file requires any modifications, the user must convert the modified XML file to a flat text file. The command below will run the Perl program ``parm/PostXMLPreprocessor.pl`` to generate the ``postxconfig*`` flat file:
+
+   .. code-block:: console
+
+      cd UPP/parm
+      # Global/GFS
+      /usr/bin/perl PostXMLPreprocessor.pl gfs/postcntrl_gfs_f00_two.xml post_avblflds.xml gfs/postxconfig-NT-gfs-f00-two.txt
+      # OR Regional/RRFS
+      /usr/bin/perl PostXMLPreprocessor.pl rrfs/rrfs_postcntrl.xml post_avblflds.xml rrfs/postxconfig-NT-rrfs.txt
+
+.. note::
+   ``PostXMLPreprocessor.pl`` must be run from the ``parm`` directory or it will produce an error. 
+
+.. attention:: 
+
+   On Ursa, the usual process for generating a new/updated ``postxconfig*.txt`` file is slightly different due to a missing XML module (see `Issue #1250 <https://github.com/NOAA-EMC/UPP/issues/1250>`_). The following workaround has been developed:
+
+   .. code-block:: console
+
+      wget https://raw.githubusercontent.com/wiki/NOAA-EMC/UPP/perl_venv_create.sh
+      chmod 755 perl_venv_create.sh
+      ./perl_venv_create.sh perl_venv
+      source perl_venv/bin/activate
+      cpanm XML::LibXML
+      cd /path/to/UPP
+      cd parm
```

Comment:
@jdkublnick Could you also turn this into a code snippet and place it also around line 253 of InputsOutputs?